README.md

```diff
@@ -1906,4 +2057,4 @@ For linters less commonly used, MegaLinters offers a plugins architecture so any
   - Run test classes and code coverage with pytest during validation GitHub Action
   - Validate descriptor YML files with json schema during build
   - Automated job to upgrade linters to their latest stable version
-<!-- mega-linter-vs-super-linter-section-end -->
+  <!-- mega-linter-vs-super-linter-section-end -->
```

Comment:
I'm not sure that build script will work well if you change the section tags, plz can you revert them ?